pkgs/applications/video/lossless-cut/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, mkYarnPackage }:
+
+mkYarnPackage rec {
+  name = "lossless-cut";
+  version = "3.23.7";
+
+  src = fetchFromGitHub {
+    owner = "mifi";
+    repo = name;
+    rev = "v${version}";
+    sha256 = "14vfmidj0m11vd1asghmcxwj102c468n6wb4swagnjzrxqh22xa3";
+  };
+
+  yarnNix = ./yarn.nix;
+  yarnLock = "${src}/yarn.lock";
+  packageJSON = "${src}/package.json";
```

Comment:
I guess, this relies on Import From Derivation, which is not allowed in nixpkgs. Seems like other packages put this file into nixpkgs directly.

Also these two lines seem to be set by default:
https://github.com/NixOS/nixpkgs/blob/db5bbef31fa05b9634fa6ea9a5afbea463da88ea/pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix#L226-L227

Reply:
I removed the three lines specifying `yarnNix`, `yarnLock` and `packageJSON` and deleted `yarn.nix`.

Reply:
I didn't mean to say that the default settings will work for nixpkgs. You might need to copy file to nixpkgs repo at least for `packageJSON` to avoid IFD.